pkg/ebpf/c/tracee.bpf.c

```diff
@@ -349,7 +359,7 @@ SEC("raw_tracepoint/sys_execve")
 int syscall__execve(void *ctx)
 {
     program_data_t p = {};
-    if (!init_program_data(&p, ctx))
+    if (!init_tailcall_program_data(&p, ctx))
```

Comment:
Are those changes to tailcall fixes?

Reply:
using the tailcall version of the `init_program_data` keeps all the fields of `p->event` untouched, which also includes the `matched_policies` value that might have been set in the calling program

Reply:
Guess they are, as this program is a tailcall from the syscall enter/exit probe.